packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Policies
+meta:
+  - name: description
+    content: An overview of what policies are and how you can use them.
+---
+## What are policies
+
+Policies help you manage access to your applications and APIs. You can restrict access based on a number of conditions such as user and group membership, device, location, or time. You can also require more authentication steps for access to sensitive applications, such as confirmation of a push notification to a mobile device or re-authentication through an SMS one-time passcode.
+
+Policies are objects that Okta evaluates when a request is made. Rules in the policies define permissions that determine whether the request is allowed or denied. Policies are stored in Okta as JSON documents.
+
+There are many possibilities for policy use:
+
+* Create authorization rules based on complex logic using conditions
+* Determine extra levels of authentication that must be performed before a specific Okta application can be accessed
+* Maintain a white-list of users and deny access based on multiple conditions
+* Change the returned scopes of the access token and add claims to it and to the ID token using [inline hooks](/docs/concepts/inline-hooks/)
+* Notify other systems through an API when a sign in happens in real-time
+* Query for user information from a database/API using scopes and add it to the user profile object
```

Comment:
Suggest changing to something like:

> Use the scopes of a token to look up user information in an external database or API, then add that data to the user's profile object

Reply:
The main thing I tripped up on here was _which_ database/API was being queried ("Okta's? Mine?")